src/components/calcite-sortable-list/calcite-sortable-list.tsx

```diff
@@ -200,6 +207,19 @@ export class CalciteSortableList {
   // --------------------------------------------------------------------------
 
   render(): VNode {
-    return <slot />;
+    const { layout } = this;
+    const layoutProp = layout == "vertical" || layout == "horizontal" ? layout : "vertical";
+    const layoutClass = layoutProp === "vertical" ? CSS.containerVertical : CSS.containerHorizontal;
```

Comment:
@driskull If you have a suggestion for a cleaner way to do this, I'd love to know it. :)

Reply:
@asangma how about this?

```ts
const layoutClass =(layout ?? "vertical") === "vertical" ? CSS.containerVertical : CSS.containerHorizontal;
```

or 

```ts
const layoutProp = layout  ?? "vertical";
```

```ts
class={{
          [CSS.container]: true,
          [CSS.containerVertical]: layoutProp === "vertical",
          [CSS. containerHorizontal]: layoutProp === "horizontal"
        }}
```

Reply:
Ah...that'd work if checking for a null value yeah?
I think our default value should handle that.

I'm concerned about an invalid value, e.g. "foo".

I guess I could redo the style to be column and add a row rule when the horizontal CSS class is added?

Reply:
Yeah that would be better. make the fallback a little easier.

```ts
class={{
          [CSS.container]: true,
          [CSS.containerVertical]: effectiveLayout !== "horizontal",
          [CSS. containerHorizontal]: effectiveLayout === "horizontal"
        }}
```

Reply:
```
const horizontal = layout === "horizontal"  ||  false;

class={{
          [CSS.container]: true,
          [CSS.containerVertical]: !horizontal,
          [CSS. containerHorizontal]: horizontal
        }}
```

Reply:
Thanks @driskull.